Exercise 4/behaviouralPattern/CommandPattern.java

```diff
@@ -0,0 +1,141 @@
+package behaviouralPattern;
+
+class Cleaner{
```

Comment:
Can have space between class name and the curly braces. For Example : Cleaner  {